lib/pulselistener.js

```diff
@@ -51,12 +52,29 @@ var retryConnect = function(connectionString, retries) {
  * {
  *   namespace:         // Namespace to prefix queues/exchanges (optional)
  *                      // defaults to `username` if given otherwise ""
- *   username:          // Username to connect with (and namespace if not given)
- *   password:          // Password to connect with
- *   hostname:          // Hostname to connect to using username/password
- *                      // defaults to pulse.mozilla.org
- *   connectionString:  // connectionString cannot be used with username,
- *                      // password and/or hostname.
+ *   hostname:          // Hostname to connect to (optional, defaults to
+ *                      // pulse.mozilla.org)
+ *   // Contact info sent to taskcluster-pulse when requesting credentials
+ *   // (required when credentials are managed by taskcluster-pulse)
+ *   contact: {         // One of:
+ *     method: 'irc',
+ *     payload: {
+ *       // Specify one of channel or user.
+ *       channel:       // Including the leading '#', e.g. '#some-channel'
+ *       user:
+ *     }
+ *   }, {               // or
+ *     method: 'email',
+ *     payload: {
+ *       address:
+ *     }
+ *   }
```

Comment:
You're missing a property: `credentials` with taskcluster credentials to passed to `new tc.Pulse({credentials})`.

I would also include an optional `baseUrl` for `new tc.Pulse({credentials, baseUrl})`, so it can be configured to work with taskcluster-pulse running locally.